Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -111,7 +115,7 @@ namespace Babylon::Polyfills::Internal
         }
     }
 
-    bool NativeCanvas::UpdateRenderTarget()
+    void NativeCanvas::UpdateRenderTarget()
     {
         if (m_dirty)
```

Comment:
`m_dirty` is never set back to true anymore, correct? Is this now just lazily creating the render target resources?

Reply:
it's set to true a few lines above when width/height has changed.